### DISCUSSION TITLE
No Access

### DISCUSSION THREAD

**Discussion by barrettepp** [aaronwmorris/indi-allsky#1213] [General]
Indi-allsky / raspberry pi question.  

My allsky camera has been up and running for a few months.  A couple weeks ago I got it running off an nvme.  Everything is working correctly except it will sometimes freeze up.  
My first issue was that I would loose access to the web page.  I would go to raspberrypi.local and there would be nothing there (yes on correct network). It would still be showing up in my router and being on and running.  I then tried to VNC into it but realized I could not do that either.   So I would go out and restart it. Upon restart I could go to raspberrypi.local and everything would be working fine and all the pictures would be there from when I could not get into it.   So the software was working on taking pictures I just could not access it.   
I could still not access it by VNC so I took it down yesterday and found out when I installed everything on the Nvme I forgot to instal VNC.  I got that installed and back up with everything working.   Was taking great photos.  I work up this morning and I could not pull up raspberrypi.local again.  I also could not pull it up via VNC.   I went outside and did a manual reboot of the machine and everything started. I could access it via both the website and VNC.  It also took a full nights worth of photos. Any idea on what could be going on or things to try.    
Also is there any way to force it to remake a timelapse.  When I manually rebooted it this morning it must have been making the timelapse so i did not get one last night and the photos are great.   

Thank you

**Comment by aaronwmorris** [comment]
It sounds like the PI is disconnecting from the network.  Is the Pi connecting via wifi?

You can regenerate timelapses by using the green `Generate` link on the Timelapse page (you have to be logged in).  You have to first delete the asset you want to generate, then generate them.  The indi-allsky capture process has to be running in order to work.

**Comment by barrettepp** [comment]
anyone have much experience with using watchdog.  
I have been reading up on setting it up with watchdog.  I have not found a lot of information on the internet so I wanted to check with all you experts to see if this looks reasonable.   
[https://defgw.wordpress.com/.../watchdog-reboot.../](https://defgw.wordpress.com/2017/10/24/watchdog-reboot-raspberry-pi-if-network-is-down/?fbclid=IwAR2EfGv0HMwVG0J3Ep7JP4vAhd0IoZi8HGCgek1BebMippeklAP0O5ZJuFM_aem_AeDinDDqQFT4uXuX2JRmZsI90F5MbpZaAlMyzgJmufCIzrwSyNXPGgczRVpDkWaXYq85TJF3gE7sTTN6bjf1KXe9)
these are the settings given on this website.  
# Watchdog ping: if unresponsive, reboot:
interface = wlan0    # use interface wlan0
ping-count = 5       # ping 5 times
ping = 192.168.1.   # ping test destination IP address
# Change default interval from 1 second to 20:
interval = 20        # perform watchdog checks every 20 seconds
Can I change this interval to something longer than 20.  I feel if I cant a lost network condition every 2 min would be fine.

**Reply by aaronwmorris** [reply]
A watchdog is a good way to go if you are having problems maintaining the wireless connection.  I use the watchdog via systemd for all of my cameras, but that does not support network testing.  I connect all of my cameras via ethernet powered via PoE, so losing connectivity is less of a problem (versus wifi).

I have not done much objective testing of this, but I did notice on my Pi3 devices that wifi became rather unstable during the heavy heat during the summer.  It is starting to warm up, so it is something to consider.
